js/utils/getBitcoinPrice.js

```diff
@@ -13,107 +13,115 @@ module.exports = function(currency, callback){
   if(currency !== 'BTC')
   {
 
+    window.btcAverages = {};
     var btPrices = [];
+    var btcAverages = {rates: {}};
 
-    //initial call
-    //bitcoin average is hitting the rate limit
-    /*
-  $.ajax({
-    method: "GET",
-    url: "https://api.bitcoinaverage.com/ticker/global/" + currency
-  })
+
+    var callBlockchain = function ()
+    {
+      $.ajax({
+        method: "GET",
+        url: "https://blockchain.info/ticker"
+      })
       .done(function (response)
       {
-        //console.log("bitcoinAverage: " + response['24h_avg']);
-        if($.isNumeric(response['24h_avg'])) {
-          btPrices.push(response['24h_avg']);
+        var blockChainCurrencies = {};
+        for (var currency in response) {
```

Comment:
Don't re-use currency as a variable name, it makes it confusing as to whether you're referring to the loop's currency var or the currency parameter that is part of the parent scope. Use something specific to this loop, like blockChainCurrency.